packages/tokens/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@juntossomosmais/atomium-tokens",
-  "version": "1.2.0",
+  "version": "2.0.0-alpha.1",
```

Comment:
Nat, não podemos commitar isso. É uma versão temporária

Reply:
@Natalie9 We need to make a rebase to remove this commit to avoid problems in CIDI

Reply:
done